OWNERS.md

```diff
@@ -71,6 +71,14 @@ without further review.
 * Otto van der Schaaf ([oschaaf](https://github.com/oschaaf)) (oschaaf@redhat.com)
 * Tim Walsh ([twghu](https://github.com/twghu)) (walsh@redhat.com)
 
+In addition to the permanent Envoy PST, we have additional temporary
+contributors from:
+
+* [Trail of Bits](https://www.trailofbits.com/) until MM/DD/YYYY.
```

Comment:
@ameily @alessandrogario @mike-myers-tob can you provide a finish date here and review? Thanks.